products/health-care/appointments/va-online-scheduling/express-care-research/express-care-conversation-guide.md

```diff
@@ -0,0 +1,111 @@
+# Express Care Conversation Guide
+
+## Moderator logistics
+
+Use [#feedback-backchannel](https://dsva.slack.com/messages/C40B45NJK/details/) in Slack for real-time feedback from observers.*
+
+*Before the session, send out the [observer instructions](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/research/during-research/howto-observer-instructions.md) to your observers (Slackbot will do this for you if you type "observer instructions" into #feedback-backchannel).*
+
+*In the Attendees section, make sure everyone except the participant is on mute*
+
+*In the Audio section, click the drop-down arrow and make sure "Play Entry/Exit Chimes" is unchecked*
+
+*Check your [screensharing setup]*
+
+*When the participant is ready, begin the session with the following intro*
+
+## Intro - 5 minutes
+
+Thanks for joining us today! My name is Melissa and I also have some colleagues on the line observing and taking notes. Today we're going to talk about scheduling Telehealth, or virtual medical, same-day appointments with a VA provider. I am going to ask you a series of questions and we'll take a look at a prototype to help my team understand how make this process easier for Veterans.
+
+Before we start, a few things I want to mention:
+
+- **This entire session should take about 45 minutes.** I want to be sure not to keep you much longer, so I may occasionally prompt you with the next question or topic.
+- **In this session, we want to hear your honest opinions.** We are not testing your ability. We just want to improve these tools to better meet Veteran's needs. I will not be offended by any opinions you express, and I welcome your feedback.
+- **You'll be interacting with a prototypes and a demo tool.** This is a demo tool that may not function exactly the way you expect. Some areas of the prototype will be clickable, and some will not. Since it's a demo, none of your actions will affect your actual VA information or benefits.
+- **If for any reason and at any time you want to stop the session, please let me know.** 
+- **Are you comfortable if I record my screen and audio as we talk today?** We use the recordings to confirm that we have captured your opinions accurately. The recordings are destroyed after we complete analysis, and none of your comments will be attributed to you directly. 
+  - If yes: **Once I start recording, I am going to confirm that you are ok with me recording this session once more.** 
+
+*Start recording.*
+
+- **I have started recording. I'd like to confirm: Are you comfortable if I record my screen the audio as we talk today?**
+
+## Section 0: Warm-up questions
+
+Before we look at the website, let’s start with a few warm-up questions.
+
+- Have you heard of "Express Care"?
+  - If YES: 
+    - What is it? 
+    - Have you used it?
+  - If NO: What do you think it might be?
+  - Prompt if needed: Is it Telehealth or in-person?
```

Comment:
have seen in prev sessions that veterans don't know the word telehealth. I might press them to define eit in their own words.

Reply:
TBH, I struggled with what to refer to it as, especially since I didn't want to use Express Care. I was thinking about having them explain it in their own words 👍great minds